testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -1724,7 +1724,19 @@ def test_set_dimensions(self):
         u.dimensions = np.array([10, 11, 12, 90, 90, 90])
         assert_allclose(u.dimensions, box)
 
+    def test_universe_kwargs(self):
```

Comment:
Because we don't use ``self`` anywhere inside? Many of the other methods don't use ``self`` anywhere inside either.